README.md

```diff
@@ -38,11 +38,12 @@ These key log messages contain a map of values that can be used by systems such
 
 ## Usage
 
-Set the environmental variables `OCR_TESSERACT_THREAD_POOL_SIZE`, `OCR_QUEUE_CAPACITY`, `LOW_CONFIDENCE_TO_LOG`, `HUMAN_LOG` and `LOGLEVEL`
+Set the environmental variables `OCR_TESSERACT_THREAD_POOL_SIZE`, `OCR_QUEUE_CAPACITY`, `LOW_CONFIDENCE_TO_LOG`, `HUMAN_LOG`, `LOGLEVEL` and `HOST_WHITE_LIST`
 
 - Run `make dev` to build JAR (versioned in target and unversioned in top level d) and run the unit tests **(using Java 11)**
 - Run `docker build -t ocr-api .` to build the docker image
-- Run `docker run -e OCR_TESSERACT_THREAD_POOL_SIZE -e OCR_QUEUE_CAPACITY -e HUMAN_LOG -e LOGLEVEL -t -i -p 8080:8080 ocr-api` to run the docker image
+- Run `docker run -e OCR_TESSERACT_THREAD_POOL_SIZE -e OCR_QUEUE_CAPACITY -e HUMAN_LOG -e LOGLEVEL -e HOST_WHITE_LIST -t -i -p 8080:8080 ocr-api` to run the docker image
```

Comment:
I think that this is in docker-chs-development, isn't it? If so, why run it standalone?

Reply:
Yeah it is on docker-chs. I quite like seeing the standalone option on repos as well as the dockerCHS option however I do see why this could cause some confusion so happy to remove it if you want. 
Also noticed that this repo doesn't have a tilt file so if that isn't a lot of work I can add it in this ticket.
The new env var will also need to be added to dockerCHS in a separate PR

Reply:
No need to remove it, just add instructions for bringing it up in dockerchs. Tiltfile - yup, why not?